app/router.js

```diff
@@ -2,12 +2,16 @@ import Ember from 'ember';
 import config from './config/environment';
 
 const Router = Ember.Router.extend({
-  location: config.locationType
+  location: config.locationType,
+  rootURL: config.rootURL
 });
 
 Router.map(function() {
   this.route('about');
   this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:rental_id' });
```

Comment:
Went with an id-based a approach to loading individual rentals as opposed to a "slug".  The thought is that this will simplify things for an intro tutorial quite a bit.